ReactiveCocoaTests/Swift/SignalSpec.swift

```diff
@@ -36,6 +36,26 @@ class SignalSpec: QuickSpec {
 				expect(signal).to(beNil())
 			}
 
+			it("should not keep signal alive indefinitely even if it has an observer") {
+				weak var signal: Signal<AnyObject, NoError>? = {
+					let signal: Signal<AnyObject, NoError> = Signal.never
```

Comment:
Can we make this explicit (e.g. `Signal.init { _ in }`) and tweak the test description to "should deallocate the signal if it never sends events"